src/package.xml

```diff
@@ -1,126 +1,130 @@
+      isExternalFile="True" externalFileFilter="GeoTIFF Files (*.tif)|*.tif|All Files (*.*)|*.*"
+      isRaster="True" />
+  </dataSheet>
+
+  <dataSheet name="outputSpatialCategory" displayName="Per Category">
+    <column name="movementTypesID" displayName="Connectivity category" dataType="Integer"
+      validationType="Datasheet" formula1="omniscape_movementTypes" allowDbNull="False" />
+    <column name="perCategoryRaster" displayName="File path" dataType="String" isExternalFile="True"
+      externalFileFilter="GeoTIFF Files (*.tif)|*.tif|All Files (*.*)|*.*" isRaster="True" />
+  </dataSheet>
+
+  <dataSheet name="outputTabularDifferences" displayName="Differences Summary">
+    <column name="movementTypesID" displayName="Connectivity category" dataType="Integer"
+      validationType="Datasheet" formula1="omniscape_movementTypes" allowDbNull="False" />
+    <column name="amountAreaDifference" displayName="Area difference" dataType="Double"
+      validationType="Decimal" validationCondition="None" />
+    <column name="percentCoverDifference" displayName="Proportion difference" dataType="Double"
+      validationType="Decimal" validationCondition="None" />
+  </dataSheet>
+
+  <dataSheet name="outputTabularChange" displayName="Transitions Summary">
+    <column name="movementTypesIDfrom" displayName="From" dataType="String" />
+    <column name="movementTypesIDto" displayName="To" dataType="String" />
+    <column name="amoutAreaChange" displayName="Area change" dataType="Double"
+      validationType="Decimal" validationCondition="None" />
+    <column name="percentCoverChange" displayName="Proportion change" dataType="Double"
+      validationType="Decimal" validationCondition="None" />
+  </dataSheet>
+
+  <dataSheet name="outputTabularJaccard" displayName="Jaccard Dissimilarity">
+    <column name="movementTypesID" displayName="Connectivity category" dataType="Integer"
+      validationType="Datasheet" formula1="omniscape_movementTypes" allowDbNull="False" />
+    <column name="dissimilarity" displayName="Dissimilarity" dataType="Double"
+      validationType="Decimal" validationCondition="None" />
+  </dataSheet>
+
+  <!--Transformers-->
+
+  <transformer
+    name="omniscapeImpactTransformer"
+    displayName="3 - Connectivity Impact Assessment"
+    transformerSpecificOutput="False"
+    isRunnable="True"
+    programName="python"
+    programArguments="omniscapeImpactTransformer.py"
+    condaEnv="omniscapeEnvironmentv2.yml"
+    condaEnvVersion="1">
+    <dataSheet name="outputTabularJaccard" type="Output" />
+    <dataSheet name="outputTabularDifferences" type="Output" />
+    <dataSheet name="outputTabularChange" type="Output" />
+    <dataSheet name="outputSpatialOverall" type="Output" />
+    <dataSheet name="differenceScenarios" type="Input" />
+    <dataSheet name="outputSpatialCategory" type="Output" />
+  </transformer>
+
+  <!--Layouts-->
+
+  <layout type="Scenario" insertAfter="omniscape_Advanced">
```

Comment:
I don't think the `insertAfter` exists anymore. This used to tell SyncroSim where to put the datasheets in the tree, but now each package has it's own tab, so this no longer does anything.